Cargo.toml

```diff
@@ -9,9 +9,11 @@ license = "MIT/Apache-2.0"
 exclude = [".gitignore"]
 
 [dependencies]
-libc = "^0.2.20"
-mio = "^0.6.4"
-tokio-core = "^0.1.4"
+libc = "0.2"
+mio = "0.6"
+bytes = "0.4"
+tokio-core = "0.1"
+tokio-io = "0.1"
```

Comment:
Can you also do the [same](https://github.com/Rufflewind/tokio-file-unix/pull/1#discussion_r113037237) for these too?